src/main/scala/dpla/ingestion3/CdlHarvesterMain.scala

```diff
@@ -36,111 +34,77 @@ object CdlHarvesterMain {
     // Parse and set the command line options
     val conf = new CdlHarvestConf(args)
     val outFile = new File(conf.outputFile.getOrElse("out"))
-    // Will never get here b/c if the API key isn't
-    // provided this will blow up Scallop validation
-    val apiKey = conf.apiKey.getOrElse("MISSING API KEY")
-    val query = conf.query.getOrElse("*:*")
-    val rows = conf.rows.getOrElse("100")
-    // Same as apiKey, this is validated at invocation.
-    val endpoint = conf.endpoint.getOrElse("MISSING ENDPOINT")
-
-    implicit val formats = DefaultFormats
+    val queryParams = Map(
+      "query" -> conf.query.getOrElse("*:*"),
+      "rows" -> conf.rows.getOrElse("100"),
+      "api_key" -> conf.apiKey.getOrElse("MISSING API KEY")
+    )
```

Comment:
I removed endpoint from the API configs and chose to hard code it while building the query URLs in each harvester since they are provider specific.